db/migrations/20181221164327-create-deleted-subscription.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('DeletedSubscriptions', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.BIGINT,
+    },
+    githubId: {
+      allowNull: false,
+      type: Sequelize.BIGINT,
+    },
+    channelId: {
+      allowNull: false,
+      type: Sequelize.STRING,
+    },
+    type: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      defaultValue: 'repo',
+    },
+    slackWorkspaceId: {
+      type: Sequelize.BIGINT,
+      allowNull: false,
+    },
+    installationId: {
+      type: Sequelize.BIGINT,
+      allowNull: true,
+      hooks: true,
+    },
+    creatorId: {
+      type: Sequelize.BIGINT,
+      allowNull: true,
+    },
+    settings: {
+      type: Sequelize.JSON,
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
+    reason: {
+      type: Sequelize.STRING,
+      allowNull: true,
+    },
+  }),
+  down: queryInterface => queryInterface.dropTable('DeletedSubscriptions'),
+};
```

Comment:
⚠️ scope creep alert ⚠️ 

Do we have any way of making sure that the columns stay in sync even when someone is making changes to the Subscriptions table?
Perhaps a test that asserts that the columns are the same (accounting for `reason`, `deleatedAt`, etc.)

Reply:
There's at least one test that validates all the fields of the subscription.

https://github.com/integrations/slack/pull/754/files#diff-d662e6c04a8d262a012ac1efa9a6f9f7R103

The `reason` column is tested in many (all?) tests regarding deleted subscriptions.

I can be more explicit about other tests where I just test the count() of deletedsubscriptions, but since all the deleted subscriptions are created the same way (with `create(subscription.dataValues)` I didn't see it as necessary.